vm/src/vm.rs

```diff
@@ -407,6 +412,11 @@ impl VirtualMachine {
         state.frozen.extend(frozen);
     }
 
+    /// Set the custom signal channel for the interpreter
+    pub fn set_user_signal_channel(&mut self, signal_rx: signal::UserSignalReceiver) {
```

Comment:
is this never set to None once it is set?

Reply:
Nope, or at least I don't see a point in doing so? It's just a configuration thing, so if someone doesn't want it they just don't call it in the first place.